examples/cgroup-rstat-flushing.yaml

```diff
@@ -0,0 +1,31 @@
+metrics:
+  counters:
+    - name: cgroup_rstat_flush_total
+      help: Total number of times cgroup rstat were flushed (recorded per level)
+      labels:
+        - name: level
+          size: 4
+          decoders:
+            - name: uint
+    - name: cgroup_rstat_locked_total
+      help: Total number of times rstat lock was obtainted and contended state
+      labels:
+        - name: contended
+          size: 4
+          decoders: # contended boolean converted to 0 and 1
+            - name: uint
+    - name: cgroup_rstat_locked_yield
```

Comment:
`cgroup_rstat_locked_yields_total`, see https://prometheus.io/docs/practices/naming/

Reply:
Thanks for the [link](https://prometheus.io/docs/practices/naming/) , as a Prometheus newbie I didn't realize that the __suffix_ had a meaning.
I now realize that general strucure is _library_name_unit_suffix_.

Jakub recommended: [Prometheus: Up & Running, 2nd Edition](https://www.oreilly.com/library/view/prometheus-up/9781098131135/)